store/tikv/gc_worker.go

```diff
@@ -43,10 +44,37 @@ type GCWorker struct {
 	lastFinish  time.Time
 	cancel      goctx.CancelFunc
 	done        chan error
+
+	session tidb.Session
```

Comment:
Data race may exists, I see both `doGC` and `StartSafePointChecker` goroutine call `loadSafePoint` related function, and they both use this session?

Reply:
Thanks, it has been changed.